phoenicis-repository/src/main/java/org/phoenicis/repository/dto/ApplicationDTO.java

```diff
@@ -72,6 +69,27 @@ public String getDescription() {
         return scripts;
     }
 
+    /**
+     * Returns the main miniature belonging to this {@link ApplicationDTO}.
+     * The main miniature is the miniature with the file name <code>main.png</code>.
+     * If no such miniature exists the first miniature of this {@link ApplicationDTO} is returned.
+     * If this {@link ApplicationDTO} contains no miniatures {@link Optional#empty()} is returned.
+     *
+     * @return An optional with the found URI, or {@link Optional#empty()} if no miniature exists
+     */
+    public Optional<URI> getMainMiniature() {
+        Optional<URI> result = this.miniatures.stream().filter(uri -> uri.getPath().endsWith("main.png")).findFirst();
```

Comment:
Isn't there an annotation or other way to specify methods that should not be serialized?
I know from other libraries (JPA, JAX-B) that such libraries normally contain ways to ignore fields/methods, I guess this should be possible for Jackson too, right?